afb-types/src/josev-types.rs

```diff
@@ -0,0 +1,494 @@
+use afbv4::prelude::*;
```

Comment:
Hmm ... Adding josev-types.rs here creates an unneeded coupling between charging and josev binding. We should think here as a generic way to handle the charging logic, whatever ISO15118 stack we use.
So you've added here one verb to set the current payment mode. Maybe just add you own enum with the two possible values EIM and PNC.
At the opposite josev binding would have to convert (if needed) from these enum to its own enum ...

Reply:
You mean adding an enum for payment option in charging-types instead of josev-types ?

Reply:
That would work yes

Reply:
Ok I agree on keeping the charging logic generic. I will replace the josev type with chmgr (ChargingMsg::Iso(IsoState)) by adding Eim and Pnc variants in it 👍

Reply:
Done